packages/calling/src/CallingClient/CallingClient.test.ts

```diff
@@ -588,85 +613,60 @@ describe('CallingClient Tests', () => {
       );
     });
 
-    it('Return a successful call object while making call', () => {
-      const call = callingClient.makeCall({address: '5003', type: CallType.URI});
-
-      expect(call).toBeTruthy();
-      expect(callingClient.getCall(call ? call.getCorrelationId() : '')).toBe(call);
-      expect(call ? call['direction'] : undefined).toStrictEqual(CallDirection.OUTBOUND);
-      call?.end();
+    it('returns undefined when there is no connected call', () => {
+      line.register();
+      line.makeCall({address: '123456', type: CallType.URI});
+      expect(callingClient.getConnectedCall()).toEqual(undefined);
     });
 
-    it('Return a successful call object while making call to FAC codes', () => {
-      const call = callingClient.makeCall({address: '*25', type: CallType.URI});
-
-      expect(call).toBeTruthy();
-      expect(call ? call['direction'] : undefined).toStrictEqual(CallDirection.OUTBOUND);
-      call?.end();
-    });
+    it('returns the connected call', () => {
+      line.register();
+      const mockCall = line.makeCall({address: '1234', type: CallType.URI});
+      const mockCall2 = line.makeCall({address: '5678', type: CallType.URI});
+      // Connected call
+      mockCall['connected'] = true;
+      mockCall['earlyMedia'] = false;
+      mockCall['callStateMachine'].state.value = 'S_CALL_ESTABLISHED';
 
-    it('Remove spaces from dialled number while making call', () => {
-      const call = callingClient.makeCall({address: '+91 123 456 7890', type: CallType.URI});
+      // Held call
+      mockCall2['connected'] = true;
+      mockCall2['held'] = true;
+      mockCall2['earlyMedia'] = false;
+      mockCall2['callStateMachine'].state.value = 'S_CALL_ESTABLISHED';
```

Comment:
Shouldn't the state here be S_CALL_HOLD ?

Reply:
Yes, it should be S_CALL_HOLD. Changed